README.md

```diff
@@ -6,9 +6,104 @@ This is a Visual Studio Code extension that supports debugging using gdb and any
 
 This extension provides a number of features that integrate into the Visual Studio Code debug environment. This includes launch types, support for the standard debug views as well as a custom viewer for memory browsing.
 
-## Launch Settings
-
-TODO
+## Prequisites
+
+External tools are expected to be present on your system depending on the intended use case.
+* **Local GDB Debug**: A GDB installation is required on your system.
+* **Remote GDB Debug**: A GDB installation is required on your system. In addition to that you must have either access to a running GDB server instance (`attach` use case), or a suitable GDB server variant must be installed to `launch` and attach to your debug target.
```

Comment:
The word `attach` is overloaded a little in the DAP area, so in this line I recommend changing attach to connect as follows.

```suggestion
* **Remote GDB Debug**: A GDB installation is required on your system. In addition to that you must have either access to a running GDB server instance (`attach` use case), or a suitable GDB server variant must be installed to `launch` and connect to your debug target.
```